classad/_expression.py

```diff
@@ -2,19 +2,19 @@
 from collections import MutableMapping
 
 import pyparsing as pp
-from typing import Any, Iterable, List, Iterator, Optional, Union
+from typing import Any, Iterable, List, Iterator, Optional, Union, Tuple
```

Comment:
``Any`` is imported but unused.

```suggestion
from typing import Iterable, List, Iterator, Optional, Union, Tuple
```